bin/tester/index.js

```diff
@@ -87,38 +87,60 @@ async function prepareAppOrBuild({
       ({ name: buildDirName } = dirSync({ unsafeCleanup: true, prefix: `${names.script}-` }));
       debug(`Building Storybook to ${buildDirName}`);
 
+      const args = [
+        '--',
+        '-o',
+        buildDirName,
+        // Make Storybook build as quiet as possible
+        ...(storybookVersion && gte(storybookVersion, STORYBOOK_CLI_FLAGS_BY_VERSION['--loglevel'])
+          ? ['--loglevel', log.level === 'verbose' ? 'verbose' : 'error']
+          : []),
+      ];
+
       const child = await startApp({
         scriptName: buildScriptName,
-        // Make Storybook build as quiet as possible
-        args: [
-          '--',
-          '-o',
-          buildDirName,
-          ...(storybookVersion &&
-          gte(storybookVersion, STORYBOOK_CLI_FLAGS_BY_VERSION['--loglevel'])
-            ? ['--loglevel', log.level === 'verbose' ? 'verbose' : 'error']
-            : []),
-        ],
-        inheritStdio: true,
+        args,
+        options: { stdio: ['pipe', 'pipe', 'pipe'] },
       });
 
       // Wait for the process to exit
       await new Promise((res, rej) => {
-        child.on('error', rej);
+        const lines = [];
+        const rl = readline.createInterface({ input: child.stderr });
+
         child.on('close', code => {
           if (code !== 0) {
-            rej(new Error(`${buildScriptName} script exited with code ${code}`));
+            console.log('');
```

Comment:
FYI I think `console.log()` has the same effect

Reply:
This will put the error message at the bottom.

Reply:
I'm just saying I think `console.log() === console.log('')` 🤷‍♂

Reply:
ah, i see 👍